### ISSUE TITLE
Do not emit TypeDoesNotContainType for interfaces

### ISSUE THREAD

**Issue by muglug** [vimeo/psalm#43]
Should not emit error

```php
interface A {
}
interface B {
    function foo();
}
function bar(A $a) {
    if ($a instanceof B) {
        $a->foo();
    }
}
```
